docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -88,28 +88,24 @@ environment around the Agents.
 
 ### Brain
 
-Brains are assets that exist in your project folder. The Ball3DAgents are connected
-to a Brain, for example : the **3DBallLearning**.
-A Brain doesn't store any information about an Agent, it just
+As of v0.6, a Brain is a Unity asset and exists within the `UnitySDK` folder. These brains (ex. **3DBallLearning.asset**) are loaded into each Agent object (ex. **Ball3DAgents**).  A Brain doesn't store any information about an Agent, it just
 routes the Agent's collected observations to the decision making process and
-returns the chosen action to the Agent. Thus, all Agents can share the same
-Brain, but act independently. The Brain settings tell you quite a bit about how
+returns the chosen action to the Agent. All Agents can share the same
+Brain, but would act independently. The Brain settings tell you quite a bit about how
 an Agent works.
 
-You can create Brain objects by selecting `Assets -> 
-Create -> ML-Agents -> Brain`. There are 3 kinds of Brains :
-The **Learning Brain** is a Brain that uses a Neural Network to take decisions.
-When the Brain is checked as `Control` in the Academy **Broadcast Hub**, the 
-external process will be taking decisions for the agents
-and generate a neural network when the training is over. You can also use the
+You can create new Brain assets by selecting `Assets -> 
+Create -> ML-Agents -> Brain`. There are 3 types of Brains. 
+The **Learning Brain** is a Brain that uses a trained neural network to make decisions.
+When the `Control` box is checked in the Brains property under the **Broadcast Hub** in the Academy, the external process will  take over decision making for the agents
```

Comment:
Maybe change this to:

When the `Control` box is checked in the Brains property under the **Broadcast Hub** in the Academy, the external process with a learning neural network will take over decision making.